.eslintrc.json

```diff
@@ -67,6 +94,8 @@
     "@typescript-eslint/no-array-constructor": "error",
     "@typescript-eslint/no-empty-interface": "error",
     "@typescript-eslint/no-explicit-any": "error",
-    "@typescript-eslint/no-parameter-properties": "error"
+    "@typescript-eslint/no-parameter-properties": "error",
+    "eslint-plugin/require-meta-docs-description": "off", // TODO: enable
+    "eslint-plugin/require-meta-type": "off" // TODO: enable
```

Comment:
i get that eslint happens to use a loose json parser that supports comments, but i'd prefer we keep this valid json and remove them 👀